source/app/ImmuneMLApp.py

```diff
@@ -15,6 +28,49 @@
 from source.util.ReflectionHandler import ReflectionHandler
 
 
+_DATE_FMT = '%d.%m.%Y %H:%M:%S'
+
+
+def hello_world(infile, rundir, outfile):
+    """Print out a politically correct greeting for immuneML.
```

Comment:
I think the 'politically correct' joke is a bit out of place, people are _very_ opinionated about this..

Reply:
can we add another joke instead? ;)

Reply:
I don't know, ask the German?

Reply:
sound like a great idea :D